### ISSUE TITLE
CSS styling of highlighted lines in code blocks adjusts the indentation

### ISSUE THREAD

**Issue by tvon** [ansible/ansible#32096]
##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME

Noticed on this page http://docs.ansible.com/ansible/latest/playbooks_blocks.html

##### ANSIBLE VERSION
n/a

##### CONFIGURATION
 n/a

##### OS / ENVIRONMENT

Shows up in current Chrome and Firefox on OSX (did not test others).

##### SUMMARY

The CSS for highlighted lines in `pre` blocks adjusts the indent of code (via padding and border) which makes yaml alignment sometimes unclear. 

In the following screenshot the `block:` line is slightly indented due to padding and a border, which makes it not-entirely-clear how indented it should be.

![screen shot 2017-10-24 at 2 52 19 pm](https://user-images.githubusercontent.com/25628/31962368-5a15d67e-b8cb-11e7-954e-5639c25f8b22.png)

Just as an example, I used Stylish to remove the border and padding for `.hll` class (the highlighted code) and changed the highlight color is a bit:  

```css
pre .hll {
    border: none;
    background-color: #f5ee9e;
    padding: 0;
}
```

![screen shot 2017-10-24 at 2 52 10 pm](https://user-images.githubusercontent.com/25628/31962472-a7a40e24-b8cb-11e7-807e-fb5fda68c83d.png)

I am not a designer so the color may be questionable, but the alignment is the important part IMO.

##### STEPS TO REPRODUCE

Visit the page

**Comment by gundalow** [comment]
@dharmabumstead May be worth looking at this as part of updating the RTD CSS that will be done as part of the TOC work.

**Comment by tvon** [comment]
The indentation issue appears to be resolved with the 2.5 docs. The highlight color is a bit difficult to make out, but that is unrelated to this issue.

![screen shot 2018-03-29 at 3 02 28 pm](https://user-images.githubusercontent.com/25628/38108293-521ac724-3362-11e8-9860-14543689f909.png)

So, closing.
